src/blog.html

```diff
@@ -0,0 +1,305 @@
+      <div class="impressions">
+        <div class="card">
+          <h1 class="impressions__title">More impressions, more conversions</h1>
+          <p class="impressions__desc">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam
+            erat, sed diam voluptua invidunt ut labore.
+          </p>
+          <button class="btn">Get Free trial</button>
+        </div>
+        <img class="impressions__img" src="/image/statistics.png" alt="" />
+      </div>
+      <!-- Logo ipsum -->
+      <div class="logoipsum-sect">
+        <img class="logoipsum-sect__image" src="/image/Logoipsum.png" alt="">
+        <img class="logoipsum-sect__image" src="/image/Logoipsum-2.png" alt="">
+        <img class="logoipsum-sect__image" src="/image/Logoipsum-3.png" alt="">
+        <img class="logoipsum-sect__image" src="/image/Logoipsum-4.png" alt="">
+        <img class="logoipsum-sect__image" src="/image/Logoipsum-5.png" alt="">
+      </div>
+      <!-- Discover section -->
+      <div class="bi-section">
+        <div class="card">
+          <p class="tag">Discover More</p>
+          <h1 class="bi-section__title">Analyze your sales and marketing leads</h1>
+          <p class="bi-section__desc">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua invidunt ut labore.
+          </p>
+        </div>
+        <div class="card">
+          <div class="bi-section__card">
+            <img src="/image/sales.png" alt="">
+            <div class="bi-section__card-desc">
+              <h1 class="bi-section__card-title">Sales Tracking</h1>
+<p class="bi-section__text">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+              eiusmod.
+            </p>
+            </div>
+            
+          </div>
+          <div class="bi-section__card">
+          <img src="/image/project.png" alt="">
+            <div class="bi-section__card-desc">
+              <h1 class="bi-section__card-title">Project Management</h1>
+<p class="bi-section__text">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+              eiusmod.
+            </p>
+            </div>
+          </div>
+          <div class="bi-section__card">
+           <img src="/image/activity.png" alt="">
+            <div class="bi-section__card-desc">
+              <h1 class="bi-section__card-title">Activity Report</h1>
+<p class="bi-section__text">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+              eiusmod.
+            </p>
+            </div>
+          </div>
+         
+        </div>
+      </div>
+      <div class="features">
+        <h1 class="title">Features</h1>
+        <div class="tri-section tri-section--features">
+     <div class="features__card-1 card">
```

Comment:
why are you naming this _features__card-1_  did you really need it if you did; then this is not a good name